backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubClient.java

```diff
@@ -29,8 +29,8 @@ public interface DataHubClient {
   UploadResponse uploadFhir(
       @RequestBody() String fhirNDJson, @RequestHeader(value = "Authorization") String authHeader);
 
-  @PostMapping(value = "/api/token")
-  TokenResponse fetchAccessToken(@SpringQueryMap Map<String, String> parameters);
+  @PostMapping(value = "/api/token", consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE)
```

Comment:
I'm struggling so hard to find doc on the usage of this annotation in the context of OpenFeign, but do we know if this will actually set the request header?

Reply:
Here's a [Baeldung link](https://www.baeldung.com/spring-cloud-post-form-url-encoded-data#2-using-map) that talks about the FeignClient and setting this header. I find it easier to understand with the examples then just the docs.